app/components/login/Login.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { createClass, Component } from 'react';
+
+const Login = createClass({
+
+  signUserIn(email, password) {
```

Comment:
You'll eventually want to create another reducer for this, with action and action-type files. Maybe the reducer can be called `user`. And, you'll want to put this action into `mapActionsToProps`. But, you can do this whenever you get there.